### PR TITLE
[Backport 7.x.x/8.x.x] Clamp & fixed position during rescale

### PR DESCRIPTION
**Pull request by skhiat**
**PR workflow guidelines**
* SRP ABV will start automatically when you open your PR
* Changes to docs and md files will **not** trigger ABV jobs 
* Consider making use of **draft PRs** if you are not 100% sure that your PR is ready for review
* ABV will restart if you add a new commit to a branch with an open PR (hence why you should consider using draft PRs)
* Adding [skip ci] (case insensitive) to the title of PRs will stop any jobs being trigger automatically - you will need to open Yamato and find your branch to run ABV

### Checklist for PR maker
- [ ] Have you added a backport label (if needed)? For example, the `need-backport-2019.3` label. After you backport the PR, the label changes to `backported-2019.3`.
- [ ] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [ ] Have you updated or added the documentation for your PR? When you add a new feature, change a property name, or change the behavior of a feature, it's best practice to include related documentation changes in the same PR.
- [ ] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

---
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?

---
### Testing status

**Manual Tests**: What did you do?
- [ ] Opened test project + Run graphic tests locally
- [ ] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/PlanarReflectionProbeEditor.cs [hunk 1]
```diff
@@ -161,7 +161,7 @@ void OnOverlayGUI(Object target, SceneView sceneView)
 
             var previewWidth = k_PreviewHeight;
             var previewSize = new Rect(previewWidth, k_PreviewHeight + EditorGUIUtility.singleLineHeight + 2, 0, 0);
-            
+
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/PlanarReflectionProbeEditor.cs]: you shouldn't have change this file

File: com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/Volume/InfluenceVolumeUI.Handles.cs [hunk 2]
```diff
@@ -102,17 +102,21 @@ static void DrawBoxHandle(SerializedInfluenceVolume serialized, Editor owner, Tr
...
                 box.size = serialized.boxSize.vector3Value;
 
                 EditorGUI.BeginChangeCheck();
                 box.DrawHull(true);
                 box.DrawHandle();
                 if (EditorGUI.EndChangeCheck())
                 {
-                    var newPosition = transform.rotation * box.center;
-                    Undo.RecordObject(transform, "Moving Influence");
-                    transform.position = newPosition;
```
Comment (RSlysz) [inline com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/Volume/InfluenceVolumeUI.Handles.cs]: Does this change have been tested with Reflection probe being child of object with position non null and transformation non identity ?

File: com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/Volume/InfluenceVolumeUI.Drawers.cs [hunk 3]
```diff
@@ -57,12 +56,19 @@ public static void SetInfluenceAdvancedControlSwitch(SerializedInfluenceVolume s
...
             s_BoxBaseHandle.monoHandle = false;
             s_BoxInfluenceHandle.monoHandle = !advanced;
             s_BoxInfluenceNormalHandle.monoHandle = !advanced;
 
+            Vector3 localSize = serialized.boxSize.vector3Value;
+            for (int i = 0; i < 3; ++i)
+            {
+                localSize[i] = Mathf.Max(Mathf.Epsilon, localSize[i]);
+            }
+            serialized.boxSize.vector3Value = localSize;
```
Comment (RSlysz) [inline com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/Volume/InfluenceVolumeUI.Drawers.cs]: This should be in the end change check:
Changing value each time it is drawn can cause infinite loop while viewing inspector of prefab asset

File: com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/Volume/InfluenceVolumeUI.Handles.cs [hunk 4]
```diff
@@ -102,17 +102,21 @@ static void DrawBoxHandle(SerializedInfluenceVolume serialized, Editor owner, Tr
             using (new Handles.DrawingScope(Matrix4x4.TRS(Vector3.zero, transform.rotation, Vector3.one)))
             {
                 box.center = Quaternion.Inverse(transform.rotation) * transform.position;
+
+                Vector3 localSize = serialized.boxSize.vector3Value;
+                for (int i = 0; i < 3; ++i)
+                {
+                    localSize[i] = Mathf.Max(Mathf.Epsilon, localSize[i]);
+                }
+                serialized.boxSize.vector3Value = localSize;
```
Comment (RSlysz) [inline com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/Volume/InfluenceVolumeUI.Handles.cs]: This should be in the end change check:
Changing value each time it is drawn can cause infinite loop while viewing inspector of prefab asset

File: com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/Volume/InfluenceVolumeUI.Handles.cs (hunk 2 above)
Comment (skhiat) [reply com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/Volume/InfluenceVolumeUI.Handles.cs]: [Tested]

File: com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/Volume/InfluenceVolumeUI.Drawers.cs (hunk 3 above)
Comment (skhiat) [reply com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/Volume/InfluenceVolumeUI.Drawers.cs]: [Fixed]
https://github.com/Unity-Technologies/ScriptableRenderPipeline/pull/6288

File: com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/Volume/InfluenceVolumeUI.Handles.cs (hunk 4 above)
Comment (skhiat) [reply com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/Volume/InfluenceVolumeUI.Handles.cs]: [Fixed]
https://github.com/Unity-Technologies/ScriptableRenderPipeline/pull/6288

**Pull request closed without merging**